jsapp/js/mixins.es6

```diff
@@ -344,7 +345,7 @@ mixins.droppable = {
     let isProjectReplaceInForm = (
       this.props.context === PROJECT_SETTINGS_CONTEXTS.REPLACE
       && router.isActive('forms')
-      && router.params.assetid !== undefined
+      && router.params.uid !== undefined
```

Comment:
@jnm Yeah this is a change to not have two ways of naming the uid in route params. Previously it was both `assetid` and `uid` mixed. We should remove `params.assetid` in the future for sure, just didn't want to do it here because it would touch too many different places :D